doc/kubernetes/modules/ROOT/pages/running/metrics/ispn_metrics.adoc

```diff
@@ -0,0 +1,87 @@
+= {ispn} Metrics
```

Comment:
`jdgserver_name` is only available in Keycloak docs. 
https://github.com/keycloak/keycloak-benchmark/blob/ae24aa8069d0421639e4716d4e7c0e6d7836301d/doc/kubernetes/modules/ROOT/partials/running/infinispan-attributes.adoc#L25

Reply:
I was on auto-pilot and didn't check the repository. Shouldn't we create this PR just against the Keycloak repository so that we don't have to migrate to the Keycloak format in a week or so?

Reply:
Seems a good topic for the Monday meeting. None of the metrics have been copied over the Keycloak project yet.

Reply:
BTW, do you prefer the table format or this one: https://www.keycloak.org/keycloak-benchmark/kubernetes-guide/latest/running/jvm/jvm_metrics

Reply:
I like the table format because it's very easy to see what type the metric is, but that KCB example is also very readable. In the Keycloak docs I think the tables used to show the various configuration options are very readable and I think a similar approach for metrics would work well.

Reply:
@ryanemerson thanks for the feedback.

Reply:
My 2 cents: If we publish it in the KCB repo first, we can ask people for feedback in the SRE SIG. We now have 90+ people there you can review / extend / comment on it once we have something published there.